### DISCUSSION TITLE
how to use my own proxy?

### DISCUSSION THREAD

**Discussion by kangsichen** [searxng/searxng#3586] [Q&A]
I need to add a proxy to the docker environment, but when I docker run -e http_proxy="xxxx"，the searxng timeout for every query.
How to fix it?

**Comment by return42** [comment]
If you don't get feedback here I recommend to discuss such topics within the SearXNG community at  [#searxng:matrix.org](https://matrix.to/#/#searxng:matrix.org) ..
